pkg/service/replay/replay.go

```diff
@@ -339,7 +339,7 @@ func (r *Replayer) RunTestSet(ctx context.Context, testSetID string, testRunID s
 
 	cmdType := utils.FindDockerCmd(r.config.Command)
```

Comment:
Instead of using this function `utils.FindDockerCmd()` Can you please take the cmdType from `r.Config.CommandType` where it is being used.